protos/flow.proto

```diff
@@ -56,6 +58,10 @@ message FlowConnectionConfigs {
   // the below two are for eventhub only
   int64 push_batch_size = 21;
   int64 push_parallelism = 22;
+
+  MappingType mapping_type = 23;
+  // if MappingType is SCHEMA, this contains the schema level mappings.
+  map<string, string> schema_mapping = 24;
```

Comment:
lets also take a boolean here: `allow_table_additions`, I think it will be useful for when people don't want to allow new tables.

Reply:
added